src/components/skills-quiz/__mocks__/react-instantsearch-dom.jsx

```diff
@@ -76,4 +88,8 @@ MockReactInstantSearch.Configure = function Configure() { return <div>CONFIGURED
 MockReactInstantSearch.Index = function Index({ children }) { return children; };
 
 // It is necessary to export this way, or tests not using the mock will fail
-module.exports = MockReactInstantSearch;
+module.exports = {
+  ...MockReactInstantSearch,
+  setFakeHits,
+  resetMockReactInstantSearch,
```

Comment:
I'm not sure it makes sense to spread `MockReactInstantSearch` here, as it should largely be acting as a default export, not named exports. Consider something like the following to keep `MockReactInstantSearch` as if it were a default export.

```tsx
module.exports = MockReactInstantSearch;
Object.assign(module.exports, {
  setFakeHits,
  resetMockReactInstantSearch,
});
```